tool/src/main/java/org/wildfly/security/tool/CredentialStoreCommand.java

```diff
@@ -108,8 +117,15 @@
     public static final String PUBLIC_KEY_STRING_PARAM = "public-key-string";
     public static final String KEY_PASSPHRASE_PARAM = "key-passphrase";
 
-    private static final List<String> filebasedKeystoreTypes = Collections.unmodifiableList(Arrays.asList("JKS", "JCEKS", "PKCS12"));
+    public static final String GENERATE_SECRET_KEY = "generate-secret-key";
+    public static final String SIZE = "size";
```

Comment:
There is already SIZE_PARAM with the same value. Is this intentional? It has a different message/description assigned - "Size (number of bytes) of the keys when generating a KeyPairCredential."

Reply:
Thank you, I have added a commit to fix this one.

This code actually started being written at the same time as the generate-key-pair code so both added the same parameter,

Reply:
Thank you for the fix, this can be resolved.